packages/caliper-ethereum/package.json

```diff
@@ -26,7 +26,7 @@
         "ethereumjs-wallet": "^0.6.3"
     },
     "devDependencies": {
-        "web3": "0.20.7",
+        "web3": "1.2.2",
```

Comment:
What about directly updating to 1.2.7?

Reply:
It's presently broken, so this change is just to revert to the way it was before it was broken. I'm not opposed to updating, but I don't think that should be prompted by this sort of breakage.